### ISSUE TITLE
Slick carousel's switch to a better carousel

### ISSUE THREAD

**Issue by jjlankinen** [otavanopisto/muikku#5143]
Open front page uses slick carousel which has quite poor WCAG support and should be replaced with different library. One candidate might be https://github.com/express-labs/pure-react-carousel#readme for it's replacement as slick has not been under active development for a long time.
